downloadDirectory/src/test/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/CellularComponentAnnotationBuilderTest.java

```diff
@@ -105,4 +105,21 @@ public void cellularComponentEmptyCompartmentReturnsNullTest() throws Exception
         assertThat(goaLines.size(), is(equalTo(1)));
         assertThat(goaLines.get(0), is(equalTo("")));
     }
+
+    @Test
+    public void cellularComponentExcludedMicrobialSpeciesReturnsArrayWithEmptyStringTest() throws Exception {
+        PowerMockito.mockStatic(GOAGeneratorUtilities.class);
+        mockProteinSet.add(mockProteinInst);
+
+        Mockito.when(GOAGeneratorUtilities.retrieveProteins(mockReactionInst)).thenReturn(mockProteinSet);
+        Mockito.when(mockProteinInst.getAttributeValue(ReactomeJavaConstants.referenceEntity)).thenReturn(mockReferenceEntityInst);
+        Mockito.when(mockProteinInst.getAttributeValue(ReactomeJavaConstants.species)).thenReturn(mockSpeciesInst);
+        Mockito.when((GOAGeneratorUtilities.isValidProtein(mockReferenceEntityInst, mockSpeciesInst))).thenReturn(true);
+        Mockito.when(((GKInstance) mockSpeciesInst.getAttributeValue(ReactomeJavaConstants.crossReference))).thenReturn(mockCrossReferenceInst);
+        Mockito.when(mockCrossReferenceInst.getAttributeValue(ReactomeJavaConstants.identifier)).thenReturn("813");
+        Mockito.when(GOAGeneratorUtilities.isExcludedMicrobialSpecies("813")).thenCallRealMethod();
+        List<String> goaLines = CellularComponentAnnotationBuilder.processCellularComponents(mockReactionInst);
+
+        assertThat(goaLines.size(), is(equalTo(0)));
```

Comment:
If you prefer, for checking a list with hamcrest, you can also do:

`assertThat(goaLines, is(empty());`